package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fundamental-ngx",
-  "version": "0.43.0-rc.4",
+  "private": true,
```

Comment:
why did you have to change that?

Reply:
Top level package version is redundant for monorepos such as this (it being outdated for months proves that point, but also recommended to be removed https://github.com/lerna/lerna/issues/2879#issuecomment-1471606448)